params/config.go

```diff
@@ -403,7 +408,12 @@ var (
 			FeeVaultAddress:           nil,
 			MaxTxPerBlock:             nil,
 			MaxTxPayloadBytesPerBlock: nil,
-			L1Config:                  &L1Config{5, common.HexToAddress("0x0000000000000000000000000000000000000000"), 0, common.HexToAddress("0x0000000000000000000000000000000000000000")},
```

Comment:
This is a good change. We could also change the line above (line 405). Previously when I added a new fork block, for instance, it was always a bit hard to find how I should update that line.

Reply:
resolved in 388456193d461957d805b809a8278e3a34b390b4